corehq/apps/locations/templates/locations/manage/locations.html

```diff
@@ -167,16 +174,18 @@ <h2>{% trans "Manage Organization Structure" %}</h2>
             name="location_search"
             style="max-width:80%;"></select>
 
+            <div style="display:inline; cursor:pointer;" data-bind="if: selected_location_id(), click: clearLocationSelection">
+            <button type="button" class="btn btn-secondary"><i class="fa fa-times" aria-hidden="true"></i></button>
+            </div>
+        </div>
+
         <span class="help-block">
             <i class="fa fa-info-circle"></i>
             To quick search for a location, write your query as "parent"/descendant. For more info, see the <a href="https://confluence.dimagi.com/display/commcarepublic/Exact+Search+for+Locations" target="_blank">Location Search</a> help page.
         </span>
 
     </div>
 
-    <div class="button_section">
-        <div data-bind="template: { name: 'button-template', if: selected_location_id(), data: selected_location}"></div>
-    </div>
```

Comment:
Nice. The way you templated this out earlier makes this change super simple and clear :)